dev/com.ibm.ws.microprofile.rest.client_fat/publish/servers/MyRESTClientServer/server.xml

```diff
@@ -12,5 +12,7 @@
 
   <!--  Required to read the remote server's port system property -->
   <javaPermission className="java.util.PropertyPermission"  name="*" actions="read" />
+  <javaPermission className="java.lang.RuntimePermission" name="getClassLoader"/>
+  <javaPermission className="java.lang.RuntimePermission" name="createClassLoader"/>
```

Comment:
This would indicate to me that your new ClassLoader logic may be missing some doPriv's and you shouldn't be adding these in, but fixing the code instead to have doPriv calls.